config-cell-cleanup.html.md.erb

```diff
@@ -39,36 +39,14 @@ Selecting the best option for disk cleanup depends on the workload that the Dieg
 
 For PCF installations that primarily run buildpack-based apps, Pivotal recommends selecting the **Routinely clean up Cell Disk space** option. The **Routinely clean up Cell Disk space** option ensures that when a new stack becomes available on a cell, the old stack is dropped immediately from the cache.
 
-For PCF installations that primarily run Docker images, or both Docker images and buildpack-based apps, Pivotal recommends selecting the **Clean up Disk space once a threshold is reached** option with a reasonable threshold.
+For PCF installations that primarily run Docker images, or both Docker images and buildpack-based apps, Pivotal recommends selecting the **Clean up Disk space once usage fills disk** option with a reasonable reserve.
 
-### <a id='choosing-a-threshold'></a> Calculating a Threshold
+### <a id='choosing-a-reserve'></a> Calculating a Reserve
 
-To calculate a realistic value when configuring the disk cleanup threshold, you must identify some of the most frequently used Docker images in your PCF installation. Docker images tend to be constructed by creating layers over existing base images. In some cases, you may find it easier to identify which base Docker images are most frequently used.  
-
-To calculate the disk cleanup threshold, do the following:
-
-1. Identify the most frequently used Docker images or base Docker images. 
-
-1. Using the Docker CLI, measure the size of those images.
-<br><br>
-For example, you may determine the most frequently used images in a test deployment are `openjdk:7`, `nginx:1.13`, and `php:7-apache`. In this case, you can run the following commands to pull the identified images locally, then measure their sizes:
-<pre class="terminal">
-$> docker pull openjdk:7
-$> docker pull nginx:1.13
-$> docker pull php:7-apache
-<br><br>
-$> docker images
-REPOSITORY       TAG          IMAGE ID          CREATED             SIZE
-php              7-apache     2720c02fc079      2 days ago          391 MB
-openjdk          7            f45207c01009      5 days ago          586 MB
-nginx            1.13         3448f27c273f      5 days ago          109 MB
-...
-</pre>
-
-1. Calculate the threshold as the sum of the frequently used image sizes plus a 15-20% buffer.
-<br><br>
-For example, using the output above, the sample threshold calculation is: (&nbsp;391&nbsp;MB&nbsp;+&nbsp;586&nbsp;MB&nbsp;+&nbsp;109&nbsp;MB&nbsp;)&nbsp;*&nbsp;1.2&nbsp;=&nbsp;1303.2&nbsp;MB
+The space reserved for other jobs will be set to a sensible default in the case that the only other component to consider is the diego-cache.
+If you are deploying with add-ons which will require space in the ephemeral disk, you may wish to increase the reserve.
```

Comment:
What do you mean by add-ons-- do you mean BOSH Add-ons like IPsec and Antivirus?

Reply:
Yes.

Reply:
hey, I deleted/resolved my other comment, but I think it would be helpful to have a harder calculation or more specific example here of how much you may want to increase. for each add-on, add xx GB? or MB?

Reply:
We think rather than duplicating docs we could just reference the open source garden-runc-release docs for configuring this value. We've opened an issue to make the calculation clearer:

https://github.com/cloudfoundry/garden-runc-release/issues/105

Reply:
After chatting with our PM about this, we think this would be better:

> The space reserved for other jobs will be set to a sensible default and we do not reccomend changing this value.
> For more information see documentation here: https://github.com/cloudfoundry/garden-runc-release/blob/develop/docs/garbage-collection.md

We really don't want people changing it because they shouldn't need to except in the weirdest of circumstances (if ever).